documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    if os.path.exists(DOWNLOAD_LOCATION):
</pre><p>I think most of this block can be replaced with <code>os.makedirs(.., exist_ok=True)</code>. Having a <em>file</em> called ~/.kaggle seems rare enough that we shouldn't need to handle it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>